components/forms/WeightForm.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+
+export default function WeightForm({ onAddWeight }) {
+  function sendForm(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const { weight, date } = Object.fromEntries(formData);
+
+    onAddWeight(weight, date);
+  }
+  return (
+    <WeightFormElement aria-label="Add weight and date" onSubmit={sendForm}>
+      <input
+        type="number"
+        name="weight"
+        min="0"
+        step="0.001"
+        aria-label="weight input"
```

Comment:
The solution with using aria-label is ok if you do not want to use the `label` element itself. If you do not want the label to be displayed another option is to use a label element and style it so that it is not displayed but will be read by screenreaders. Think about my suggestion, but for now it is good. :)